src/main/java/racing/domain/Car.java

```diff
@@ -8,6 +8,10 @@
     public Car( String name){
         this.name = name;
     }
+    public Car( String name, int totalMeter){
+        this.name = name;
+        this.allMeter = totalMeter;
+    }
```

Comment:
파라미터 앞부분은 공백으로 띄어져 있네요 😭 
공백을 제거해주시면 좋을 것 같아요! 그다음 파라미터가 끝나는 부분과 블록 시작 "{" 사이에는 공백을 넣어주시면 좋을 것 같아요 :)
```suggestion
    public Car(String name) {
        this.name = name;
    }
    public Car(String name, int totalMeter) {
        this.name = name;
        this.allMeter = totalMeter;
    }
```